onoff.js

```diff
@@ -5,8 +5,13 @@ const debounce = require('lodash.debounce');
 const Epoll = require('epoll').Epoll;
 
 const GPIO_ROOT_PATH = '/sys/class/gpio/';
+
+// fs reads return char data
 const ZERO = Buffer.from('0');
 const ONE = Buffer.from('1');
```

Comment:
If `GPIO_ONE` is renamed to `HIGH` (see below) then `ONE` should be renamed to `HIGH_BUF`.